pkg/bundle/internal/ssa_client/patch.go

```diff
@@ -17,10 +17,86 @@ limitations under the License.
 package ssa_client
 
 import (
+	"encoding/json"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	coreapplyconfig "k8s.io/client-go/applyconfigurations/core/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	trustapi "github.com/cert-manager/trust-manager/pkg/apis/trust/v1alpha1"
+	trustapiac "github.com/cert-manager/trust-manager/pkg/applyconfigurations/trust/v1alpha1"
 )
 
+func GenerateBundlePatch(
+	bundlePatch *trustapiac.BundleApplyConfiguration,
+) (*trustapi.Bundle, client.Patch, error) {
+	if bundlePatch == nil || bundlePatch.Name == nil {
```

Comment:
@erikgb can you also check that the patch does not have a namespace?

Reply:
Also, can we change this back to `GenerateBundleStatusPatch` and only patch the status?

Reply:
> can you also check that the patch does not have a namespace?

Yes, we can do that, makes sense.

> Also, can we change this back to GenerateBundleStatusPatch and only patch the status?

Why? A patch is a patch.